pkg/output/tui/progress_test.go

```diff
@@ -0,0 +1,72 @@
+//go:build !race
+
+// TODO: there is a race condition in the progress code that needs to be fixed
+//       somehow
```

Comment:
@dsimansk This is ugly. 

There's [a race condition](https://gist.github.com/cardil/5762ff149f10fe1d76ddadc8b01c9f8d), but I think it's in the bubblegum library itself, rather than my code.

Maybe we should investigate that, and at least open an issue on bubblegum project?!

Reply:
I'm fine to keep it here for now. Given we either capture it in issue here at least or preferably on bubblegum lib with further details.

Reply:
Done in 7dd518ec94f8e3ff46f338a84d034ac6d94576a2

Reply:
I'll try doing a reproducer versus the [github.com/charmbracelet/bubbles/progress](https://pkg.go.dev/github.com/charmbracelet/bubbles/progress).